tricolour/apps/tricolour/app.py

```diff
@@ -398,6 +375,8 @@ def _main(args):
         # Visibilities from the dataset
         vis = getattr(ds, data_column).data
         if args.subtract_model_column is not None:
+            warnings.warn("-smc arg is depreciating."
+                          "See -dc arg for expressions")
```

Comment:
Replace with "--subtract-model-column argument is deprecated. Use --data-column instead"